eda-env/Lib/site-packages/parso/python/diff.py

```
@@ -0,0 +1,886 @@
+        self._replace_tos_indent = None
+        self._keyword_token_indents = {}
+        # print('start', line_offset + 1, indents)
+        for token in tokens:
+            # print(token, indents)
+            typ = token.type
+            if typ == DEDENT:
+                if len(indents) < initial_indentation_count:
+                    # We are done here, only thing that can come now is an
+                    # endmarker or another dedented code block.
+                    while True:
+                        typ, string, start_pos, prefix = token = next(tokens)
```

Pylint (3.3.7)

[refactor] Do not raise StopIteration in generator, use return statement instead

(R1708)